app/view_models/admin/auctions_index_view_model.rb

```diff
@@ -1,6 +1,12 @@
 class Admin::AuctionsIndexViewModel < Admin::BaseViewModel
-  def auctions
-    Auction.all.order(started_at: :desc).map do |auction|
+  attr_reader :auctions
+
+  def initialize(auctions: Auction.all.order(started_at: :desc))
```

Comment:
not this exact one, was just trying to make this view mode more consistent with `AuctionsIndexViewModel` (non-admin)